lib/wallaby/query.ex

```diff
@@ -98,24 +98,26 @@ defmodule Wallaby.Query do
           | :option
           | :select
           | :file_field
+          | :attribute
   @type attribute_key_value_pair :: {String.t(), String.t()}
   @type selector ::
           String.t()
-          | :attribute_key_value_pair
+          | attribute_key_value_pair()
   @type html_validation ::
           :bad_label
           | :button_type
           | nil
   @type conditions :: [
-          count: non_neg_integer,
-          text: String.t(),
+          count: integer,
+          minimum: integer,
+          maximum: integer,
```

Comment:
```suggestion
          count: non_neg_integer,
          minimum: non_neg_integer,
          maximum: non_neg_integer,
```

Reply:
I double checked dialyzer on the repro project locally and it doesn't affect the fix so it should be fine :)